packages/ra-ui-materialui/src/list/filter/FilterButton.spec.tsx

```diff
@@ -23,25 +24,18 @@ describe('<FilterButton />', () => {
         filterValues: {},
     };
 
-    describe('filter button', () => {
-        it('should not be rendered, if all filters are already being displayed', () => {
-            const { queryByText } = render(<FilterButton {...defaultProps} />);
-            expect(queryByText('ra.action.add_filter')).toBeNull();
-        });
-    });
```

Comment:
Removed because if we don't have any filter to show, then some filtering has been applied and we should show the button to allow users to persist it